src/interfaces/coral_web/src/components/Conversation/Header.tsx

```diff
@@ -132,20 +116,30 @@ export const Header: React.FC<Props> = ({ conversationId, isStreaming }) => {
           </Text>
         </span>
         <span className="flex items-center gap-x-2 py-4 pl-4 md:pl-0">
-          <KebabMenu
-            className={cn('md:hidden', { hidden: !conversationId })}
-            items={menuItems}
-            anchor="left start"
-          />
+          <KebabMenu className="md:hidden" items={menuItems} anchor="left start" />
           <IconButton
-            iconName="trash"
-            onClick={handleDelete}
```

Comment:
can we no longer delete chats? :')

Reply:
it was removed from designs, looks like only possible from conversation list panel ;(